### ISSUE TITLE
multipart/form-data upload lacks boundary

### ISSUE THREAD

**Issue by davisford** [swagger-api/swagger-editor#666]
I'm trying to document a simple api in the editor to upload a text file.  On the backend, I have a golang server that receives the post.  Unfortunately, the go standard library is not liking what it receives when I attempt to upload a file through the editor.  It is throwing the error: ["no multipart boundary param in Content-Type"](https://github.com/golang/go/blob/master/src/net/http/request.go#L324) when it is attempting to parse the multipart form.

I'm wondering why the [boundary](http://www.w3.org/Protocols/rfc1341/7_2_Multipart.html) is missing from the request?

Testing on latest Chrome.  Here's the spec of the api I am testing:

`````` yaml
  /orgs/{orgId}/accounts/restore:
    parameters:
      - $ref: '#/parameters/orgId'
    post:
      consumes: ['multipart/form-data']
      parameters:
        - name: accounts.txt
          in: formData
          description: |
            This should be `text/plain` looking like this:
              ```
              700000000170
              700000001668
              700000002284
              <etc...>
              ```
          type: file
          required: true
      responses:
        202:
          description: Accepted
        400:
          $ref: '#/responses/400'
        default:
          $ref: '#/responses/500'
``````

_update_: I tested the api using Advanced Rest Client Chrome extension and it works as expected, so it does appear to be something amiss in the swagger editor multipart/form-data implementation.

**Comment by mohsen1** [comment]
Somehow related to https://github.com/swagger-api/swagger-editor/issues/599

File upload component is kind of hacky in editor. It needs a proper refactoring.

**Comment by ivanhuay** [comment]
+1

**Comment by saharj** [comment]
It should be fixed by #1141 . Please let us know if you still face the issue.
